spec/webview-spec.js

```diff
@@ -321,27 +343,13 @@ describe('<webview> tag', function () {
 
   describe('disablewebsecurity attribute', () => {
     it('does not disable web security when not set', async () => {
-      const jqueryPath = path.join(__dirname, '/static/jquery-2.0.3.min.js');
```

Comment:
Can we delete jquery-2.0.3.min.js from our tests now?

Reply:
There are still a few places using it, but I think we should be able to rework those places to remove this file.